app/components/location-map.js

```diff
@@ -2,12 +2,14 @@ import { inject as service } from '@ember/service';
 import Component from '@ember/component';
 
 export default Component.extend({
-  maps: service(),
+  classNames: ['map-container'],
+  mapElement: service(),
 
   didInsertElement() {
     this._super(...arguments);
-    let location = this.location;
-    let mapElement = this.maps.getMapElement(location);
-    this.$('.map-container').append(mapElement);
+    this.mapElement.getMapElement(this.location).then((mapElement) => {
+      this.$().append(mapElement);
```

Comment:
```suggestion
      this.element.appendChild(mapElement);
```

Reply:
Since we weren't using jQuery APIs elsewhere in the PR

Reply:
fixed